apps/ledger-live-mobile/src/families/celo/Delegations/index.tsx

```diff
@@ -0,0 +1,446 @@
+  }, [onNavigate, vote, account]);
+
+  const onVote = useCallback(() => {
+    onNavigate({
+      route: NavigatorName.CeloVoteFlow,
+      screen: votes?.length === 0 ? ScreenName.CeloVoteStarted : ScreenName.CeloVoteSummary,
+      params: {},
+    });
+  }, [onNavigate]);
+
+  const onCloseDrawer = useCallback(() => {
+    setVote();
+  }, []);
+
+  const getValidatorName = useCallback(
+    (vote: CeloVote): string => {
+      const validatorInfo = validatorGroups.find(validator => validator.address === vote.validatorGroup) ||
+      fallbackValidatorGroup(vote?.validatorGroup);
+      return validatorInfo.name;
+  }, [vote],
+  );
+
+  const onOpenExplorer = useCallback(
+    (address: string) => {
+      const url = getAddressExplorer(
+        getDefaultExplorerView(account.currency),
+        address,
+      );
+      if (url) Linking.openURL(url);
+    },
+    [account.currency],
+  );
+
+  const data = useMemo<DelegationDrawerProps["data"]>(() => {
+    const v = vote;
+    let status = null;
+    if (v != null) {
+      status = voteStatus(v);
+    }
+    
+    return v
+      ? [
+          {
+            label: t("celo.delegation.validatorGroup"),
+            Component: (
+              <LText
+                numberOfLines={1}
+                semiBold
+                ellipsizeMode="middle"
+                style={[styles.valueText]}
+                color="live"
+              >
+                {getValidatorName(v) ?? ""}
+              </LText>
+            ),
+          },
+          {
+            label: t("delegation.validatorAddress"),
+            Component: (
+              <Touchable
+                onPress={() => onOpenExplorer(v.validatorGroup)}
+                event="DelegationOpenExplorer"
+              >
+                <LText
+                  numberOfLines={1}
+                  semiBold
+                  ellipsizeMode="middle"
+                  style={[styles.valueText]}
+                  color="live"
+                >
+                  {v.validatorGroup}
+                </LText>
+              </Touchable>
+            ),
+          },
+          {
+            label: t("delegation.delegatedAccount"),
+            Component: (
+              <LText
+                numberOfLines={1}
+                semiBold
+                ellipsizeMode="middle"
+                style={[styles.valueText]}
+                color="live"
+              >
+                {account.name}{" "}
+              </LText>
+            ),
+          },
+          {
+            label: t("cosmos.delegation.drawer.status"),
+            Component: (
+              <LText
+                numberOfLines={1}
+                semiBold
+                ellipsizeMode="middle"
+                style={[styles.valueText]}
+                color="live"
+              >
+                {!!(status === "active") && <> <CheckCircle color={colors.green } size={14} style={styles.icon}/> <Trans i18nKey={"celo.revoke.vote.active"} /> </>}
```

Comment:
Curious, what's the difference between `status === "active"` and `!!(status === "active")` here?

Reply:
A cool trick I recently discovered, instead of using 

`condition ? do something : null` you can achieve the same with:
` !! condition && do something` saving you the extra `: null`. Maybe doesn't improve readability but I dig it.